### PR TITLE
fix: linting on commit

### PR DESCRIPTION
**Pull request by danethurber**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,17 +31,18 @@
       "scripts/create-component/package-template/**/*"
     ],
     "linters": {
+      "**/*.css": [
+        "prettier-eslint --write",
```
Comment (danethurber) [inline package.json]: format css files

File: package.json [hunk 2]
```diff
@@ -31,17 +31,18 @@
...
+        "git add"
+      ],
       "**/*.js": [
         "prettier-eslint --write",
+        "eslint",
```
Comment (danethurber) [inline package.json]: i was confused. `prettier-eslint` just formats the file. we can use `eslint` to do the linting

File: package.json [hunk 3]
```diff
@@ -31,17 +31,18 @@
...
         "git add"
       ],
       "package.json": [
         "format-package -w",
         "git add"
-      ],
-      "packages/**/*.css": [
-        "node_modules/.bin/prettier --single-quote --no-semi --write",
```
Comment (jaketrent) [inline package.json]: Without the single quote and no semi flags, are those getting read from somewhere else that's committed to the project?  We should get a .prettierrc in here, but we don't have one yet.

Comment (jaketrent) [reply package.json]: I am blind. :)

**Pull request merged**